unstructured_inference/models/yolox.py

```diff
@@ -13,14 +13,42 @@
 from pdf2image import convert_from_path
 
 from unstructured_inference.inference.layout import DocumentLayout, LayoutElement, PageLayout
-from unstructured_inference.models import _get_model_loading_info
 from unstructured_inference.visualize import draw_bounding_boxes
+from unstructured_inference.models.base import UnknownModelException
+import json
+from typing import Tuple, Dict
+from huggingface_hub import hf_hub_download
+
+
+def get_model_loading_info(model: str) -> Tuple[str, Dict[int, str]]:
```

Comment:
Is this new?

Reply:
Used to have the _get_model_loading_info name in __init__.py, @qued / @cragwolfe mentioned that that file should keep empty